src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/requirements.txt

```diff
@@ -8,3 +8,4 @@ astroid~=2.1.0
 six~=1.11.0
 aws-sam-cli==0.22.0
 pip==19.1.1
+tenacity==6.0.0
```

Comment:
have not use this but it looks like a decent way to handle retry logic, thoughts @sbkok ?

Reply:
Mmm, I haven't used this library before. But this looks promising.